flake.nix

```diff
@@ -0,0 +1,89 @@
+{
+  description = "Nix flake for zkLLVM";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs";
+    flake-utils.url = "github:numtide/flake-utils";
+    nix-3rdparty = {
+      url = "github:NilFoundation/nix-3rdparty";
+      inputs = {
+        nixpkgs.follows = "nixpkgs";
+        flake-utils.follows = "flake-utils";
+      };
+    };
+    nil-crypto3 = {
+      url = "https://github.com/NilFoundation/crypto3?submodules=0&rev=caf78b83692b39f504c4ee74a3834a4c38662a77";
```

Comment:
Maybe we should use a bit cleaner syntax for that?

```nix
nil-crypto3 = {
    url = "github:NilFoundation/crypto3";
    rev = "caf78b83692b39f504c4ee74a3834a4c38662a77";
    inputs = { ... };
};
```

`submodules=0` and `type=git` are probably no longer needed (although I'm not sure about type).

It would be a bit easier to read I guess. But I'm not 100% sure that my code is gonna work.

Reply:
done